### PR TITLE
revert to exisiting way of removing workers

### PR DESCRIPTION
**Pull request by ankitkumarr**
cc: @kashimiz

### CODE REVIEW

File: build/BuildSteps.cs [hunk 1]
```diff
@@ -583,5 +588,17 @@ private static List<string> GetV3PackageList()
 
             return packageList;
         }
+
+        private static void RemoveLanguageWorkers(string outputPath)
```
Comment (fabiocav) [inline build/BuildSteps.cs:592]: I realize this is a tactical fix, but if we're not adding tests here, or intend to change the approach, can we please have an issue filed and linked for follow up?

**Review by fabiocav — APPROVED** [review]

File: build/BuildSteps.cs (hunk 1 above)
Comment (ankitkumarr) [reply build/BuildSteps.cs:592]: Yes absolutely. I will be opening an issue with proper details for a follow up.

**Comment by Francisco-Gamino** [comment]
Hi @ankitkumarr -- should we also revert these?

1) https://github.com/Azure/azure-functions-core-tools/blob/v3.x/build/BuildSteps.cs#L126
2) https://github.com/Azure/azure-functions-core-tools/blob/v3.x/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj#L147

**Comment by ankitkumarr** [comment]
Hi @Francisco-Gamino, I don't think so. That's for the nupkg that we started creating for core tools if I understand correctly. I think we can leave that as is for now. @kashimiz can confirm.

**Comment by Francisco-Gamino** [comment]
> Hi @Francisco-Gamino, I don't think so. That's for the nupkg that we started creating for core tools if I understand correctly. I think we can leave that as is for now. @kashimiz can confirm.

Thanks @ankitkumarr  for clarifying.

File: build/BuildSteps.cs (hunk 1 above)
Comment (ankitkumarr) [reply build/BuildSteps.cs:592]: https://github.com/Azure/azure-functions-core-tools/issues/2380

**Pull request merged**